### ISSUE TITLE
Tradie not working on linux

### ISSUE THREAD

**Issue by eastwood** [jameslnewell/tradie#15]
Tried and tested a few different instances including:
1. Ubuntu 14.04 from VirtualBox
3. Amazon EC2 instance

In all three cases, running the following commands:
1. `npm install -g tradie`
2. `tradie`

Results in:
```
Error: Cannot find module '../tradie'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/usr/lib/node_modules/tradie/dist/bin/tradie.js:12:15
```

At first I thought this was a vagrant/virtualbox problem, but after running on amazon I suspect otherwise. Perhaps there is some slight OS difference going on here?

**Comment by eastwood** [comment]
```

Error: Cannot find module '../tradie'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/usr/lib/node_modules/tradie/dist/bin/tradie.js:12:15)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
[ec2-user@ip-172-31-37-72 tradie]$ ls
API.md  dist  example  node_modules  package.json  README.md  src  test
[ec2-user@ip-172-31-37-72 tradie]$ cd /usr/lib/node_modules/tradie/
[ec2-user@ip-172-31-37-72 tradie]$ ls
API.md  dist  node_modules  package.json  README.md
[ec2-user@ip-172-31-37-72 tradie]$ tree dist
dist
├── bin
│   ├── dist
│   │   ├── index.css
│   │   └── index.js
│   ├── tradie.js
│   └── tradie.js.map
├── cmd
│   ├── build.js
│   ├── build.js.map
│   ├── bundle.js
│   ├── bundle.js.map
│   ├── bundle-scripts.js
│   ├── bundle-scripts.js.map
│   ├── bundle-styles.js
│   ├── bundle-styles.js.map
│   ├── clean.js
│   ├── clean.js.map
│   ├── init.js
│   ├── init.js.map
│   ├── lint.js
│   ├── lint.js.map
│   ├── test.js
│   └── test.js.map
├── lib
│   ├── autoprefixer-stream.js
│   ├── autoprefixer-stream.js.map
│   ├── color-stream.js
│   ├── color-stream.js.map
│   ├── createDirectory.js
│   ├── createDirectory.js.map
│   ├── createFile.js
│   ├── createFile.js.map
│   ├── createJsonFile.js
│   ├── createJsonFile.js.map
│   ├── defaultTemplate.js
│   ├── defaultTemplate.js.map
│   ├── executePlugins.js
│   ├── executePlugins.js.map
│   ├── getConfig.js
│   ├── getConfig.js.map
│   ├── loadPlugins.js
│   ├── loadPlugins.js.map
│   ├── logger.js
│   ├── logger.js.map
│   ├── minify-stream.js
│   ├── minify-stream.js.map
│   ├── readConfig.js
│   ├── readConfig.js.map
│   ├── readConfig.test.js
│   ├── readConfig.test.js.map
│   ├── readMochaOptions.js
│   ├── readMochaOptions.js.map
│   ├── requireExtension.js
│   ├── requireExtension.js.map
│   ├── requireExtension.test.js
│   ├── requireExtension.test.js.map
│   ├── scripts
│   │   ├── bundle.js
│   │   ├── bundle.js.map
│   │   ├── clean.js
│   │   ├── clean.js.map
│   │   ├── createBundle.js
│   │   ├── createBundle.js.map
│   │   ├── createBundler.js
│   │   ├── createBundler.js.map
│   │   ├── defaults.js
│   │   ├── defaults.js.map
│   │   ├── init.js
│   │   ├── init.js.map
│   │   ├── lint.js
│   │   ├── lint.js.map
│   │   ├── test.js
│   │   └── test.js.map
│   ├── size-stream.js
│   ├── size-stream.js.map
│   ├── styles
│   │   ├── bundle.js
│   │   ├── bundle.js.map
│   │   ├── clean.js
│   │   ├── clean.js.map
│   │   ├── defaults.js
│   │   ├── defaults.js.map
│   │   ├── init.js
│   │   └── init.js.map
│   ├── TradieError.js
│   ├── TradieError.js.map
│   ├── uglify-stream.js
│   └── uglify-stream.js.map
├── test
│   ├── getConfig.js
│   └── getConfig.js.map
├── Tradie.js
└── Tradie.js.map

7 directories, 86 files
[ec2-user@ip-172-31-37-72 tradie]$
[one-nib] 0:nvim- 1:ssh* 2:bash                                                                                  "hf0006317.nibdom.com." 11:24 05-Apr-16
```

**Comment by jameslnewell** [comment]
There used to be a `Tradie.js` file and I probably didn't clean my `./dist/` dir before re-building and publishing so OSX used the original case and not the newer, lowercase.

Fixed in `0.7.4`.
